default/python/AI/FleetUtilsAI.py

```diff
@@ -14,22 +14,22 @@
 __designStats = {}
 
 
-def stats_meet_reqs(stats, reqs):
+def stats_meet_reqs(stats, requirements):
     """Check if (fleet) stats meet requirements.
 
     :param stats: Stats (of fleet)
     :type stats: dict
-    :param reqs: Requirements
-    :type reqs: dict
+    :param requirements: Requirements
+    :type requirements: dict
     :return: True if requirements are met.
     :rtype: bool
     """
     try:
-        for key in reqs:
-            if stats.get(key, 0) < reqs[key]:
+        for key in requirements:
+            if stats.get(key, 0) < requirements[key]:
                 return False
         return True
-    except:
+    except KeyError:
         return False
```

Comment:
That try-except seems redundant given we only look up keys that we get from the keys() method in one dict and use ```get``` to look those up in the other dict.